src/sentry/api/bases/organization_events.py

```diff
@@ -38,7 +47,10 @@ def get_snuba_filter(self, request, organization, params=None):
 
     def get_snuba_params(self, request, organization, check_global_views=True):
         with sentry_sdk.start_span(op="discover.endpoint", description="filter_params"):
-            if len(request.GET.getlist("field")) > MAX_FIELDS:
+            if (
+                len(request.GET.getlist("field") + self.get_equation_list(organization, request))
```

Comment:
nit: would it be better to take the sum of the lengths rather than constructing a list that contains both fields and equations then taking the length?

Reply:
yup good catch, adding is O(k) while length is O(1)
```python
python3 -m timeit -v -s "a=[1]*(10**5); b=[2]*(10**5)" "len(a+b)"
10 loops -> 0.0112 secs
100 loops -> 0.0979 secs
1000 loops -> 0.907 secs
raw times: 0.967 0.941 0.834
1000 loops, best of 3: 834 usec per loop

python3 -m timeit -v -s "a=[1]*(10**5); b=[2]*(10**5)" "len(a) + len(b)"
10 loops -> 6.28e-06 secs
100 loops -> 2.05e-05 secs
1000 loops -> 0.000187 secs
10000 loops -> 0.00186 secs
100000 loops -> 0.0197 secs
1000000 loops -> 0.187 secs
10000000 loops -> 1.73 secs
raw times: 1.77 1.5 1.48
10000000 loops, best of 3: 0.148 usec per loop
```